05_stringer/starfallz/main_test.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMainFunction(t *testing.T) {
+	t.Run("Test main to return string values of IPAddr({127, 0, 0, 1}) with proper formatting", func(t *testing.T) {
+		var buf bytes.Buffer
+		out = &buf
+
+		main()
+
+		expected := strconv.Quote("127.0.0.1")
+		actual := strconv.Quote(buf.String())
+
+		if expected != actual {
```

Comment:
If you use the testify library, it would be as simple as assert.Equal(expected, actual, msg) - one line instead of 3 lines of code.

Reply:
Yeah I tried testify but I noticed that lab6 is the only one that explicitly say to use testify so I figured best to kept it plain vanilla for all labs until lab 6 so at least I get the hang of doing it without external dependencies till then.

Reply:
Not exactly, lab06 ask you to use `testify/suite`, which is not the same as `testify/assert` which is commonly used. I did what you did in lab4, and Julia commnented that I should consider using testify/assert, and it does make the code bit shorter.

Reply:
I would recommend using testify, however don't feel the need to change it since your tests are all passing. If you do the later labs, you'll probably want to use it more since it significantly shortens the test code.